python/src/trezorlib/cli/cardano.py

```diff
@@ -37,9 +37,10 @@ def cli():
     required=True,
     help="Transaction in JSON format",
 )
-@click.option("-p", "--protocol-magic", type=int, default=1)
+@click.option("-p", "--protocol-magic", type=int, default=764824073)
+@click.option("-N", "--network-id", type=int, default=0)
```

Comment:
please use default values from `cardano.PROTOCOL_MAGICS` / `NETWORK_IDS`
(now you have a mainnet proto magic but testnet network id?)

Reply:
perhaps add a `--testnet` for convenience to set both at once?

Reply:
Done [here](https://github.com/trezor/trezor-firmware/pull/1112/commits/8c4a5c8a733d9d991405e5ee1c4f9d9bfe9895bf). Also with the `--testnet` option.